assignment/tool/src/webapp/js/assignmentsByStudent.js

```diff
@@ -0,0 +1,72 @@
+(() => {
+
+// Include datatables dependencies
+window.includeWebjarLibrary('datatables');
+window.includeWebjarLibrary('datatables-rowgroup');
+
+// Assignments By Students "global" namespace
+window.ABS = {
```

Comment:
I think you should change the name of this, tbh. Nobody will know what ABS means when they're inspecting in the console. It could be "Antilock Braking System" :). I've been trying to namespace stuff with sakai.assignments, or something similar.

Reply:
I thought "Adrian Benefits Sakai" was a quite clear acronym, but you are right, I'll namespace it a bit better.